install/local/addQTIExamples.php

```diff
@@ -15,18 +17,29 @@
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  *
- * Copyright (c) 2013-2014 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ * Copyright (c) 2013-2020 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
  *
  */
 
+use oat\oatbox\service\ServiceManager;
 use oat\tao\model\TaoOntology;
 
-$testClass = taoQtiTest_models_classes_QtiTestService::singleton()->getRootClass();
-$file = dirname(__FILE__) . DIRECTORY_SEPARATOR . 'qtiv2p1Examples.zip';
+$testService = ServiceManager::getServiceManager()->get(taoQtiTest_models_classes_QtiTestService::class);
+$testClass = $testService->getRootClass();
 
-try {
-    $report = taoQtiTest_models_classes_QtiTestService::singleton()->importMultipleTests(new core_kernel_classes_Class(TaoOntology::TEST_CLASS_URI), $file);
-} catch (Exception $e) {
-    common_Logger::e('An error occured while importing QTI Test Example. The system reported the following error: ' . $e->getMessage());
-    throw $e;
+foreach (new DirectoryIterator(__DIR__) as $fileInfo) {
```

Comment:
do we still need move this inside try-catch,
is it possible that the DirectoryIterator will fail ?